lolstaticdata/championrates/__main__.py

```diff
@@ -27,28 +27,33 @@ def main():
 }}
 &variables={{"language":"en","role":"ALL","region":"world","queue":420,"tier":"PLATINUM_PLUS","patch":"{patch}"}}
 """.format(patch=patch.name)
-    data = requests.get("https://flash.blitz.gg/graphql?query=" + urllib.parse.quote(query, safe="/()=&")).json()["data"]["lolChampionsListOverview"]
+    data2 = requests.get("https://apix1.op.lol/tierlist/7/?lane=default&patch=11.8&tier=platinum_plus&queue=420&region=all").json()
```

Comment:
This url is hardcoded by patch so it will not be accurate in the future.

Reply:
I don't know what you're talking about